src/openrct2/windows/top_toolbar.c

```diff
@@ -3056,14 +3059,15 @@ void top_toolbar_init_view_menu(rct_window* w, rct_widget* widget) {
 	gDropdownItemsArgs[10] = STR_HEIGHT_MARKS_ON_LAND;
 	gDropdownItemsArgs[11] = STR_HEIGHT_MARKS_ON_RIDE_TRACKS;
 	gDropdownItemsArgs[12] = STR_HEIGHT_MARKS_ON_PATHS;
+	gDropdownItemsArgs[14] = STR_VIEW_CLIPPING;
 
 	window_dropdown_show_text(
 		w->x + widget->left,
 		w->y + widget->top,
 		widget->bottom - widget->top + 1,
 		w->colours[1] | 0x80,
 		0,
-		13
+		15
```

Comment:
Some top toolbar menus are already done in this way.  Since this PR adds a new entry to the view menu anyway I'll update this as suggested.